### PR TITLE
Update docs and changelog for v2.5.0

### PR DESCRIPTION
**Pull request by corycaywood**
# Description
Updated changelog and also added some additional information about throwing errors to a mini app.

# Checklist
- [x] I have read the [contributing guidelines](../CONTRIBUTING.md).
- [ ] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [x] I ran `./gradlew check` without errors

### CODE REVIEW

**Review by minh-rakuten — APPROVED** [review]
👍

File: miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/MiniAppMessageBridge.kt [hunk 1]
```diff
@@ -69,7 +69,10 @@ abstract class MiniAppMessageBridge {
     @VisibleForTesting
     internal fun createBridgeExecutor(webViewListener: WebViewListener) = MiniAppBridgeExecutor(webViewListener)
 
-    /** Get provided id of mini app for any purpose. **/
+    /**
+     * Get provided id of mini app for any purpose.
+     * You can also throw an [Exception] from this method to pass an error message to the mini app.
```
Comment (khairul-alam-licon) [inline miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/MiniAppMessageBridge.kt:74]: Is `getUniqueId()` throwing any exception?

**Review by khairul-alam-licon — APPROVED** [review]
Looks good 👍 
Just a qus.

File: miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/MiniAppMessageBridge.kt (hunk 1 above)
Comment (corycaywood) [reply miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/MiniAppMessageBridge.kt:74]: There is a case where the Host App wants to throw an error to the mini app when they fail to generate the Unique ID. For example, the Unique ID could be based on the User's login and they need to fetch it from an API, so the API request could fail.

**Pull request merged**